### PR TITLE
[optimus-cpp] New port: ID hashing and Obfuscation using Knuth's Algorithm for C++

### PR DESCRIPTION
**Pull request by kafeg**
new port: https://github.com/kafeg/optimus-cpp

### CODE REVIEW

File: ports/optimus-cpp/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,17 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO kafeg/optimus-cpp
+    REF 0.1.0
+    SHA512 387b1da32a653d2026d884749a69932ed47929afa92c66843a654735cdc7572f033bb64abd6553db24f03f9566e225f054234e1ec9ff177051aff3bb9ceea1ed
+    HEAD_REF main
+)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH ${SOURCE_PATH}
```
Comment (PhoebeHui) [inline ports/optimus-cpp/portfile.cmake:10]:
```suggestion
    SOURCE_PATH "${SOURCE_PATH}"
```

File: ports/optimus-cpp/portfile.cmake [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+)
+
+vcpkg_cmake_install()
+vcpkg_copy_pdbs()
+
+file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
+file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
```
Comment (PhoebeHui) [inline ports/optimus-cpp/portfile.cmake:17]:
```suggestion
file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
```

File: ports/optimus-cpp/vcpkg.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "optimus-cpp",
+  "version": "0.1.0",
+  "description": "ID hashing and Obfuscation using Knuth's Algorithm for C++",
```
Comment (PhoebeHui) [inline ports/optimus-cpp/vcpkg.json:4]:
```suggestion
  "description": "ID hashing and Obfuscation using Knuth's Algorithm for C++",
  "homepage": "https://github.com/kafeg/optimus-cpp",
```

**Review by PhoebeHui — CHANGES_REQUESTED** [review]
@kafeg, thanks for your PR!

File: ports/optimus-cpp/portfile.cmake (hunk 1 above)
Comment (kafeg) [reply ports/optimus-cpp/portfile.cmake]: Than you for mentions!

File: ports/optimus-cpp/vcpkg.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  "homepage": "https://github.com/kafeg/optimus-cpp",
+  "supports": "!uwp",
+  "dependencies": [
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    },
+    {
+      "name": "vcpkg-cmake-config",
```
Comment (Osyotr) [inline ports/optimus-cpp/vcpkg.json:13]: This is not used by the port. I was looking into project's `CMakeLists.txt` and could not find proper installation. Have you tested static builds and linux builds?

Comment (kafeg) [reply ports/optimus-cpp/vcpkg.json]: Thank you, removed `vcpkg-cmake-config` dependency.

**Review by PhoebeHui — APPROVED** [review]
LGTM, thanks for your updates!

**Comment by BillyONeal** [comment]
Thanks for the new port!

**Pull request merged**